### DISCUSSION TITLE
Question on custom image

### DISCUSSION THREAD

**Discussion by arhanjain** [maximmax42/Discord-CustomRP#157] [General]
Hey guys,

I had a quick question on how you are serving your images to the Discord RPC client. I'm not too proficient with C#, so I was hoping you could give me a quick explanation on how it works. Are you feeding the RPC client the local path of the user specified file, or perhaps hosting it on a local port and giving the URL to the RPC client? Thanks for your time, any help is appreciated. (sorry for creating an issue for this, i forgot discussions existed)

**Comment by maximmax42** [comment]
Presence API only accepts images that are either uploaded as assets for an app on discord.com/developers portal, or a direct URL to an image hosted anywhere on the internet. In first instance you pass a name of an asset to API, in second - just the URL. Local files aren't supported.

**Reply by arhanjain** [reply]
I see, so is CustomRP hosting the images chosen by users on something like AWS bucket? I'm working on something that involves images hosted on google photos, but the problem is the Presence API only takes links up to 256 characters if im not wrong. How did you guys get around this problem.

**Reply by maximmax42** [reply]
No, I don't use a custom hosting, users are uploading the images themselves, either to dev portal, or somewhere else.
